features/steps/steps.py

```diff
@@ -17,13 +17,42 @@
     contains_string,
 )
 
-from features.environment import create_uat_image
+from features.environment import (
+    create_instance_with_uat_installed,
+    capture_container_as_image,
+)
 from features.util import SLOW_CMDS, emit_spinner_on_travis, nullcontext
 
 from uaclient.defaults import DEFAULT_CONFIG_FILE, DEFAULT_MACHINE_TOKEN_PATH
 
 
-CONTAINER_PREFIX = "ubuntu-behave-test-"
+CONTAINER_PREFIX = "ubuntu-behave-test"
+IMAGE_BUILD_PREFIX = "ubuntu-behave-image-build"
+IMAGE_PREFIX = "ubuntu-behave-image"
+
+
+def add_test_name_suffix(context, series, prefix):
+    pr_number = os.environ.get("UACLIENT_BEHAVE_JENKINS_CHANGE_ID")
+    pr_suffix = "-" + str(pr_number) if pr_number else ""
+    is_vm = bool(context.config.machine_type == "lxd.vm")
+    vm_suffix = "-vm" if is_vm else ""
+    time_suffix = datetime.datetime.now().strftime("-%s%f")
+
+    name = "{prefix}{pr_suffix}{vm_suffix}-{series}{time_suffix}".format(
```

Comment:
I think we can remove this var

Reply:
Ah yep, I had pulled that out as a var while debugging and forgot to delete - thanks for catching it!